web/src/scripts/containers/TabbedEditor.jsx

```diff
@@ -157,6 +157,11 @@ class TabbedEditor extends Component {
 
     // TODO consider also using command+t (which currently conflicts with a CM command)
     openFileSearch: 'command+p',
+    nextTab: 'command+]',
+    prevTab: 'command+[',
+    openInNewTab: 'command+alt+\\',
+    focusNextGroup: 'command+k command+right',
```

Comment:
Conflicts with code folding in Atom/Sublime with default keybindings. I'm fine with this for now though. Let's just figure out if we need to switch based on keybindings at some point.